compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -928,8 +928,9 @@ impl Visitor<'tcx> for Validator<'mir, 'tcx> {
                     && tcx.lookup_stability(callee).map_or(false, |s| s.level.is_unstable());
                 if callee_is_unstable_unmarked {
                     trace!("callee_is_unstable_unmarked");
-                    // We do not use this scheme for intrinsics, as intrinsics have no other way
-                    // to get marked `const` but to have a `rustc_const_(un)stable` attribute.
+                    // We do not use `const` modifiers for intrinsic "functions", as intrinsics are
+                    // `extern` funtions, and these have way to get marked `const`. So instead we
```

Comment:
```suggestion
                    // `extern` funtions, and these have no way to get marked `const`. So instead we
```
At least I assume that's what you mean.